### PR TITLE
Admin improvements

### PR DESCRIPTION
**Pull request by jefflau**
Removed status, added filters for admin panel

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr111-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr111-app-wearekickback.surge.sh

**Review by hiddentao — CHANGES_REQUESTED** [review]
Status column is no longer showing up in CSV export.

**Comment by hiddentao** [comment]
i'll do the changes needed as i need to add in the new participant statuses anyway

**Comment by jefflau** [comment]
I left status out on purpose as I didn't think they were relevant to CSV

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr111-app-wearekickback.surge.sh

File: src/components/SingleEvent/EventParticipants.js [hunk 1]
```diff
@@ -36,7 +37,7 @@ class EventParticipants extends Component {
 
   handleSearch = search => {
     this.setState({
-      search: search || ''
+      search: search.toLowerCase() || ''
```
Comment (hiddentao) [inline src/components/SingleEvent/EventParticipants.js]: should be `(search || '').toLowerCase()`

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr111-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr111-app-wearekickback.surge.sh

**Review by hiddentao — APPROVED** [review]

**Pull request merged**